### ISSUE TITLE
 Email: ITI Comments on First Public Working Draft of W3C Accessibility Guidelines (WCAG) 3.0 (5 - Clear Words)

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#447]
[Comment from Email](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/0012.html): 
from [PDF](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/att-0012/ITI_Comments_on_WCAG_3.0_First_Public_Working_Draft_20210224_1_.pdf)

* Guidelines need to be testable/measurable based on normative information. In some new guidelines such as 7.2 Clear Words this does not appear to be testable based on normative information. In the Methods the tool Simplewriter is listed. There is no attribution for this tool (who owns it?) Instead of recommending a specific tool, WCAG should specify the criteria that should be tested, so others can incorporate it into their current tools or look for a publicly available tool.

**Comment by jmcsorle** [comment]
Draft Response:
<hr>
Thank you for your suggestions. In a future  draft of WCAG3 we will clarify that that tool is just a “proof-of-concept” and list the underlying rules so they are transparent and so that better tools can be developed.

**Comment by rachaelbradley** [comment]
#627 addresses this issue.
